### ISSUE TITLE
Can someone please increase the font selection for visually impaired users 

### ISSUE THREAD

**Issue by jasonsimpson** [aeharding/voyager#430]
$ git diff main
diff --git a/src/features/settings/appearance/TextSize.tsx b/src/features/settings/appearance/TextSize.tsx
index ed7f526..20db25d 100644
--- a/src/features/settings/appearance/TextSize.tsx
+++ b/src/features/settings/appearance/TextSize.tsx
@@ -65,7 +65,7 @@ export default function TextSize() {
             ticks
             snaps
             min={0.8}
-            max={1.6}
+            max={3.6}
             step={0.1}
             value={fontSizeMultiplier}
             onIonInput={(e) => {

You might even go to 4.6 and adjust the step level.

**Comment by jasonsimpson** [comment]
Font SIZE

**Comment by aeharding** [comment]
@jasonsimpson I am implementing this now. Please be aware that layout may appear broken with text sizes that large.

**Comment by jasonsimpson** [comment]
> @jasonsimpson I am implementing this now. Please be aware that layout may appear broken with text sizes that large.

Thank you for this.
I am aware this impacts other aspects of the UI. (Like missing full community name in the feed for each post. That could be solved by placing the community name on its own line.) 
As I was testing this, I wonder if it makes sense to add the "..." to the lower right corner of each comment, so that it does not become hidden when the top level span text expands due to font size.
